### PR TITLE
Adds content to README

### PR DESCRIPTION
**Pull request by zamith**
This is a first draft of the README, but it documents how to use crystal deps (which I believe is not documented) and how to use the helpers from users and timeline.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -6,3 +6,49 @@ A library to access the Twitter API using [Crystal][]
...
+In your terminal run:
+
+```bash
+crystal deps
+```
+
+This will get the latest code from this github repository and copy it to a `libs` directory. All that's left is to require it:
+
+```crystal
+require "./libs/twitter/twitter"
```
Comment (asterite) [inline README.markdown]: You actually just need to do `require "twitter"` as "libs" is in the default crystal path.

Comment (zamith) [inline README.markdown]: :+1:

Thanks.

**Comment by kenta-s** [comment]
@zamith 

I agree that the README should have how to use crystal deps. However, since the PR was opened long time ago, the contents need to be updated a bit.

First, `Projectfile` things should be replaced with `shard.yml`
Second, `require 'twitter'` should be replaced with `require twitter-crystal`

Do you still have enough motivation to do that? Thank you :)

**Comment by kenta-s** [comment]
I did it by myself, but I took most of your new README. That was so helpfull :) thanks

**Pull request closed without merging**